diff --git a/src/Interfaces/Standalone/hcoi_standalone_mod.F90 b/src/Interfaces/Standalone/hcoi_standalone_mod.F90
index bbf54c8..fb3a25e 100644
--- a/src/Interfaces/Standalone/hcoi_standalone_mod.F90
+++ b/src/Interfaces/Standalone/hcoi_standalone_mod.F90
@@ -2783,7 +2783,7 @@ CONTAINS
        Name = 'DRY_TOTN'
        CALL ExtDat_Set( HcoState,     ExtState%DRY_TOTN,                     &
                         TRIM( Name ), RC,                   FIRST=FIRST     )
-       IF ( RC == HCO_SUCCESS ) THEN
+       IF ( RC /= HCO_SUCCESS ) THEN
           ErrMsg = 'Could not find quantity "' // TRIM( Name )            // &
                 '" for the HEMCO standalone simulation!'
           CALL HCO_Error( ErrMsg, RC, ThisLoc )
@@ -2797,7 +2797,7 @@ CONTAINS
        Name = 'WET_TOTN'
        CALL ExtDat_Set( HcoState,     ExtState%WET_TOTN,                     &
                         TRIM( Name ), RC,                   FIRST=FIRST     )
-       IF ( RC == HCO_SUCCESS ) THEN
+       IF ( RC /= HCO_SUCCESS ) THEN
           ErrMsg = 'Could not find quantity "' // TRIM( Name )            // &
                    '" for the HEMCO standalone simulation!'
           CALL HCO_Error( ErrMsg, RC, ThisLoc )
